domain_access/src/DomainAccessManager.php

```diff
@@ -41,6 +41,7 @@ public function __construct(DomainLoaderInterface $loader, DomainNegotiatorInter
    * @inheritdoc
    */
   public function getAccessValues(EntityInterface $entity, $field_name = DOMAIN_ACCESS_FIELD) {
+    /** @var FieldableEntityInterface $entity */
```

Comment:
Right - but EntityInterface does not have a get() method - as far as I can tell, the only two extensions of EntityInterface that have a get() method are FieldableEntityInterface and ConfigEntityInterface - and it's pretty clear we are talking about the former here. I was not, however, able to convince myself that only FieldableEntities could ever get passed in. So I opted to try not to change the behavior.

Possibly the most correct thing to do would be to make the signature more strict. But that would risk causing errors. The safer thing to do would be to have a test on $entity to ensure that was fieldable (or at least had a get method) before calling get on it. Currently, I think this will fail somewhat badly if it gets an entity that is not fieldable.